src/MixedQuoter.sol

```diff
@@ -279,6 +278,58 @@ contract MixedQuoter is IMixedQuoter, IPancakeV3SwapCallback {
         return amountIn;
     }
 
+    function quoteMixedExactOutput(
+        address[] calldata paths,
+        bytes calldata actions,
+        bytes[] calldata params,
+        uint256 amountOut
+    ) external override returns (uint256 amountIn) {
```

Comment:
If we decide to implement mix swap, we'd better make sure the arguments between the quoter and router match up

Reply:
ok, good point.
Now just follow quoteMixedExactInput arguments.
If we decide to support mixed swap in universal router , i will update this